app/apis/auth.py

```diff
@@ -0,0 +1,66 @@
+# app/apis/auth.py
+"""Weconnect entry point"""
+
+from flask import Flask, jsonify
+from flask_restful import Resource, Api
+from flask_restful.reqparse import RequestParser
+
+app = Flask(__name__)               # Create Flask WSGI appliction
+api_v1 = Api(app, prefix="/api/v1")  # Wrap the app in Api
+
+# users list of user dictionary objects
+users = []
```

Comment:
When you have to pick a user from the list, how will you go about it?

Reply:
I added a get_user function that checks if the user exists